.pre-commit-config.yaml

```diff
@@ -0,0 +1,16 @@
+repos:
+-   repo: https://github.com/ambv/black
+    rev: 19.3b0
```

Comment:
how will we go about maintaining these versions? would it be possible to put a "most recent" here?

Reply:
we can put a stable for black but there is no stable tag for the other tools. Honestly, I think it is not that relevant to care about the versions here.

Reply:
ok